opencti-platform/opencti-front/src/private/components/common/lists/Filters.js

```diff
@@ -1691,63 +1714,58 @@ class Filters extends Component {
       R.sortWith([R.ascend(R.prop('label'))]),
     )(entityTypes);
     return (
-      <React.Fragment>
```

Comment:
Si je comprends bien, on supprime React.Fragment puisqu'on a qu'un seul enfant imbriqué ?

Reply:
Yes, react fragment is just a way to wrap multi component without extra div.
So if only one, its useless